src/api/operations.ts

```diff
@@ -251,6 +253,7 @@ export const drawEllipse = (options: {
   y: number | PDFNumber;
   xScale: number | PDFNumber;
   yScale: number | PDFNumber;
+  rotate: Rotation;
```

Comment:
This must be marked as optional or it will be a breaking change.
```suggestion
  rotate?: Rotation;
```